Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/branch.py

```diff
@@ -69,14 +70,34 @@ def branch_point(cls, repository):
 
         return commit
 
+    @classmethod
+    def to_branch_name(cls, value):
+        result = ''
+        for c in string_utils.decode(value):
+            if c in [u'-', u' ', u'\n', u'\t']:
+                result += '-'
```

Comment:
Does this mean we may have multiple '-' if we have multiple spaces?

Reply:
Yes. Which seems ok, it feels like multiple consecutive spaces in a bug title is probably a mistake to being with.